chevah/compat/tests/normal/test_system_users.py

```diff
@@ -46,8 +46,7 @@ def test_getHomeFolder_posix(self):
         # For buildslave, home folder is in srv.
         if mk.username == 'buildslave':
             self.assertEqual(u'/srv/' + mk.username, home_folder)
-        elif mk.username == 'runner' and self.os_name == 'osx':
-            # We are on macOS GitHub actions.
+        elif self.os_name == 'osx':
```

Comment:
I think that here we need a helper like `self.ci_name` or something like that to detect over which CI do you run the tests.

If env GITHUB_ACTIONS="true" then we have GHA.

We should also detect Travis and buildbot

Reply:
But why? `/Users` is the usual home on macOS. The exception for `buildslave` is already there…

Reply:
let me know if you need help... for buildbot, the env marker is   BUILDBOT=true

Reply:
I am using the github action hack in chevah/server and would help to have a standardinze way to detect the CI system in the tests...

yes... the mk.username == 'buildslave' is an ugly hack.

with a code like `elif self.ci_name == 'gha'` the code could be understood without the extra need for a comment like `# We are on macOS GitHub actions.`

The idea is to have the code speak for itself and organize the code and name the variables and helpers in such a way that you don't need a comment.

makes sense?

Reply:
> I am using the github action hack in chevah/server and would help to have a standardinze way to detect the CI system in the tests...

Fine by me, but for this PR my goals were minimal. Minor drive-by fixes like adding Travis CI are fine, but let's not get carried away too much.


> yes... the mk.username == 'buildslave' is an ugly hack.

Ugly, but needed for as long as we use Buildbot. Don't have an issue with that though.


> with a code like `elif self.ci_name == 'gha'` the code could be understood without the extra need for a comment like `# We are on macOS GitHub actions.`

But that comment is gone, I have removed that special case. There was no need for it anyway and it only broke something else, namely macOS builds everywhere else.